pkg/v1/tkg/constants/clusterclass_mapping.go

```diff
@@ -279,7 +279,8 @@ const (
 
 	SPEC = "spec"
 
-	TopologyClassIncorrectValueErrMsg = "input cluster class file, attribute spec.topology.class has no value or incorrect value or not following correct naming convension"
+	TopologyClassIncorrectValueErrMsg                = "input cluster class file, attribute spec.topology.class has no value or incorrect value or not following correct naming convention"
+	ClusterResourceWithoutTopologyNotSupportedErrMsg = "Input file contains Cluster resource which doesn't have ClusterClass specified. Passing Cluster resource without ClusterClass specification is not supported"
```

Comment:
Nit: the first error message starts with a lower-case (`input`) while the new one start with uppercase (`Input`).

I don't yet know how to trigger the error, so I don't have an opinion on which format is best but I was thinking we should be consistent.